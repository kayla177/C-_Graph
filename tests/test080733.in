ENTITY E2O01NIM name_E2O0 paper
ENTITY W2WZ22QF name_W2WZ institution
ENTITY BT27ZH9J name_BT27 conference
ENTITY 3HPW808I name_3HPW conference
ENTITY SEIFN0PW name_SEIF author
ENTITY O0UTL390 name_O0UT conference
ENTITY UE5RR6S5 name_UE5R author
ENTITY 47F258HW name_47F2 author
ENTITY JLUXHXIA name_JLUX journal
ENTITY UOHY1SD8 name_UOHY conference
RELATIONSHIP W2WZ22QF affiliated_with UOHY1SD8 6.8
RELATIONSHIP JLUXHXIA cited_by W2WZ22QF 2.1
RELATIONSHIP 3HPW808I cited_by UE5RR6S5 3.4
RELATIONSHIP BT27ZH9J cited_by W2WZ22QF 1.1
RELATIONSHIP SEIFN0PW collaborated_with 47F258HW 0.6
RELATIONSHIP UE5RR6S5 collaborated_with W2WZ22QF 8.9
RELATIONSHIP UOHY1SD8 collaborated_with SEIFN0PW 3.5
RELATIONSHIP E2O01NIM authored_by JLUXHXIA 0.8
RELATIONSHIP 3HPW808I presented_at JLUXHXIA 3.2
RELATIONSHIP BT27ZH9J collaborated_with UOHY1SD8 10.0
RELATIONSHIP 3HPW808I published_in O0UTL390 9.6
RELATIONSHIP JLUXHXIA published_in BT27ZH9J 7.4
RELATIONSHIP E2O01NIM authored_by UOHY1SD8 6.3
RELATIONSHIP BT27ZH9J presented_at 47F258HW 8.2
RELATIONSHIP O0UTL390 presented_at JLUXHXIA 8.9
HIGHEST
DELETE O0UTL390
PRINT UOHY1SD8
HIGHEST
DELETE 47F258HW
PRINT JLUXHXIA
DELETE JLUXHXIA
DELETE UOHY1SD8
PATH BT27ZH9J W2WZ22QF
HIGHEST
EXIT
