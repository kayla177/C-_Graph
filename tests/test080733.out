success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
E2O01NIM 3HPW808I 42.2
success
W2WZ22QF SEIFN0PW BT27ZH9J E2O01NIM
UE5RR6S5 47F258HW 33.9
success
W2WZ22QF E2O01NIM 3HPW808I BT27ZH9J
success
success
BT27ZH9J W2WZ22QF 1.1
BT27ZH9J 3HPW808I 13.4
